docs/pages/architecture/registry.rst

```diff
@@ -51,6 +53,7 @@ To validate an event signature, the following checks must be performed:
 4. Was the certificate valid at the time of signing (*IssuedAt*)?
 5. Is the owning entity of the certificate (e.g. a vendor or organization) the one we expected to sign the certificate (see *Owner check* in the table below)?
 6. Is the JWS signed using the RS256 algorithm?
+7. Is the used RSA key of sufficient length (>=2048 bits)?
 7. Is the JWS signed with the private key belonging to the public key in the certificate?
```

Comment:
Does ReST handle numbering automatically, otherwise this must be an 8?

Reply:
Fixed.